lib/python/qmk/path.py

```diff
@@ -50,8 +50,9 @@ def keymap(keyboard_name):
 
         keyboard_folder = keyboard_folder.parent
 
-    logging.error('Could not find the keymaps directory!')
-    raise NoSuchKeyboardError('Could not find keymaps directory for: %s' % keyboard_name)
+    keyboard_folder = keyboard(keyboard_name) / 'keymaps'
+    logging.debug('Could not find the keymaps directory, creating as %s', keyboard_folder)
+    keyboard_folder.mkdir(parents=True)
```

Comment:
I'm not entirely convinced that we want to just create directories.
Looking at the build failures, I think this will just mask the problem.

Imho, the CLI should catch the exception and handle it more gracefully and we should not build targets without `keymaps` dir to begin with.

Reply:
What problem do you think it's masking?

Reply:
Like trying to build targets without a `keymaps` dir. :D
I think we can safely ignore these, as a revision dir will have the keymaps.

Reply:
I mean, I would love to ignore these, but they're not buildable on the API. I want them to be buildable. That means we have to resolve this somehow, and I don't think forcing a keymaps directory into each of the currently broken boards is a good solution either.

Reply:
I ended up backing out these changes in favor of #11334.